src/app/item-page/full/full-item-page.component.html

```diff
@@ -1,6 +1,14 @@
 <div class="container clarin-shadow remove-padding clarin-home-page-borders clarin-cut-bottom-2 clarin-cut-top-1 pb-2" *ngVar="(itemRD$ | async) as itemRD">
   <div *ngIf="itemRD?.hasSucceeded" @fadeInOut>
     <div *ngIf="itemRD?.payload as item">
+      <div class="card alert-info mb-3" *ngIf="fromSubmissionObject">
+        <div class="card-header font-weight-bold">
+          {{'item.page.users.note' | translate}}
+        </div>
+        <div class="card-body">
+          {{ item?.firstMetadataValue('local.submission.note') }}
+        </div>
+      </div>
```

Comment:
are we sure it will not be shown to user? Is thee `fromSubmissionObject` sufficient condition to not show it to others than the reviewer and admin?

Reply:
The anonymous user cannot have an access to that workflow Item. I think it is not restricted, but there is almost no chance the anonymous user could find it and however he will find it he will see only some note from the submitter.